### PR TITLE
Fix missing translations: move strings to components

### PR DESCRIPTION
**Pull request by sk7**
GRAPE-19579

### CODE REVIEW

File: src/actions/newConversation.js [hunk 1]
```diff
@@ -160,12 +158,7 @@ const loadMembershipPeople = () => (dispatch, getState) => {
       if (!people.length && !peopleQuery) {
         dispatch({ type: types.HANDLE_NO_PEOPLE_LEFT_TO_JOIN })
       } else if (peoplePage === 1 && results.length) {
-        dispatch(
-          handlePeopleResults([
-            { text: 'People you already have a conversation with' },
-            ...results,
-          ]),
-        )
+        dispatch(handlePeopleResults([{ separator: true }, ...results]))
```
Comment (AleshaOleg) [inline src/actions/newConversation.js:161]: Maybe better, `isSeparator`?

Comment (sk7) [reply src/actions/newConversation.js]: yes!

Comment (sk7) [reply src/actions/newConversation.js]: @AleshaOleg pushed changes

Comment (AleshaOleg) [reply src/actions/newConversation.js]: looks good :)

**Pull request merged**